CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 ## Changelog:
 
+### 1.15.4 - 2024-02-25
+- zend-application
+  - update dependencies according to usage (#185)
+- zend-auth
+  - update dependencies according to usage (#186)
+- zend-controller
+  - update dependencies according to usage (#188)
+- zend-form
+  - fix instantiating filters with options via `addFilter()` in php 8.0+ (#194)
+- zend-json
+  - update dependencies according to usage (#189)
+- zend-layout
+  - update dependencies according to usage (#190)
+- zend-mail
+  - update dependencies according to usage (#191)
+- zend-view
+  - updated `HeadScript` to allow for proper html5 syntax creation (#103)
```

Comment:
should use this syntax: `([#103])` for to use reflinks:
- https://github.com/zf1s/zf1/pull/195/files#r1503172305